doc/source/whatsnew/v0.25.1.rst

```diff
@@ -129,7 +129,7 @@ Reshaping
 ^^^^^^^^^
 
 - A ``KeyError`` is now raised if ``.unstack()`` is called on a :class:`Series` or :class:`DataFrame` with a flat :class:`Index` passing a name which is not the correct one (:issue:`18303`)
--
+- Bug in ``.merge_asof(tolerance=tolerance)`` could not merge :class:`Timedelta` objects (:issue:`27642`)
```

Comment:
use ``:meth:`merge_asof` ``, and you can mention when passing the ``tolerance`` kwarg